app/views/shared/_comment_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_with model:([post, comment]), html: { class: 'comment-form'} do |form|%>
+  <h1>Add a comment</h1>
+  <%=form.label "Enter Comment", class: 'comment-field-label'%>
+  <%=form.text_area :text, class: 'comment-field'%>
+  <%=form.submit "Comment", class: 'common-btn'%>
+<%end%>
```

Comment:
- [ ] Kindly make sure to create a new file `new.html.erb` in the `views/comments/` folder, you can have the choice of referring to the `_comment_form.html.erb` partial there or create a new form inside the `new.html.erb`